src/modules/mc_pos_control/mc_pos_control_tests/mc_pos_control_tests.cpp

```diff
@@ -77,7 +77,7 @@ McPosControlTests::~McPosControlTests()
 
 bool McPosControlTests::cross_sphere_line_test(void)
 {
-	MulticopterPositionControl	*control = {};
+	McPosControl		*control = {};
```

Comment:
I know you didn't change the tests, but does this actually work? A few lines below control->cross_sphere_line() is called on nothing?
@AndreasAntener?